examples/preprocessors__typescript-webpack/cypress/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+	"strict": true,
+    "baseUrl": "../../../node_modules", // Point this to the root `node_modules` of your project. Most likely `../node_modules`
+    "target": "es5",
+    "lib": ["es5", "dom"],
+    "types": ["cypress"],
+  },
+  "include": [
+    "**/*.ts"
+  ]
+}
```

Comment:
@bahmutov I'll leave it up to you if you want all the comments. I removed for aesthetics (to make it obvious what is important to change)

Reply:
sure, not objections